File: src/IM.Production.UI/src/models/dtos/factory.contract.dto.ts
export class FactoryContractDto {
    /// Source Cusomer info
    public sourceCustomerLogin: string;
    /// Destination Cusomer info
    public destinationCustomerLogin: string;
    //Имя материала человеческим языком
    public materialKey: string;
    /// Количество материала, уже поставленное по контракту.
    public totalCountCompleted: number;
    /// Общая сумма на закупку/продажу, прошедшую по контракту.
    public totalSumm: number;
}

File: src/IM.Production.WebApp/Dtos/FactoryContractDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace IM.Production.WebApp.Dtos
{
    public class FactoryContractDto
    {
        /// Source Cusomer info
        public string SourceCustomerLogin { get; set; }
        /// Destination Cusomer info
        public string DestinationCustomerLogin {get; set; }
        /// Имя материала человеческим языком
        public string MaterialKey { get; set; }
        /// Количество материала
        public int TotalCountCompleted { get; set; }
        /// Общая сумма на закупку/продажу, прошедшую по контракту.
        public int TotalSumm { get; set; }
    }
}
